framework/src/userobject/RadialAverage.C

```diff
@@ -37,6 +37,11 @@ RadialAverage::validParams()
   params.addClassDescription("Perform a radial equal weight average of a material property");
   params.addRequiredParam<std::string>("material_name", "Name of the material to average.");
   params.addRequiredParam<Real>("radius", "Cut-off radius for the averaging");
+  params.addParam<Real>(
+      "padding",
+      0.0,
+      "Padding for communication. This gets added to the radius when determining which QPs to send "
+      "to other processors. It should be of the order of the maximum element size.");
```

Comment:
But `padding` is zero (default) in the tests?

Reply:
Yeah, I think it will mostly be needed for inhomogeneous element sizes

Reply:
Are you sure we even need the padding? It seems like we're already going on the safe side by using the processor bounding boxes. Wouldn't that catch all possible needed communications? The way the comment is written, users will think they need to set this to something nonzero, which doesn't seem to be the case.

I'd say we should just remove this. If it is actually needed, it should be range checked to be >0, and the description should be revised to say that it's fine to be 0 for most cases.

Reply:
Yeah, we might need it still. There is a radius search of qps around boundary nodes. That might need a wee bit of padding for some element geometries.